src/main/java/games/strategy/engine/data/CompositeRouteFinder.java

```diff
@@ -107,7 +107,7 @@ private static Route AssembleRoute(final Territory start, final Territory end,
     return new Route(routeTers);
   }
 
-  private HashMap<Territory, Integer> CreateScoreMap() {
+  private HashMap<Territory, Integer> createScoreMap() {
```

Comment:
The implementation type is probably not needed.
We should prefer returning just the interface type instead, e.g. Map<Territory, Integer>